docs/content/theme/theme-palette.md

```diff
@@ -27,101 +27,169 @@ These variables are global hooks to override colors, fonts, spacing and more thr
+| `--pfe-theme--font-family--heading` | "Overpass", Overpass, Helvetica, helvetica, arial, sans-serif |
+| `--pfe-theme--font-family--code` | "Overpass Mono", Consolas, Monaco,  Andale Mono , monospace |
+| `--pfe-theme--font-size--heading--alpha` | 32px |
+| `--pfe-theme--font-size--heading--beta` | 24px |
+| `--pfe-theme--font-size--heading--gamma` | 21px |
+| `--pfe-theme--font-size--heading--delta` | 18px |
+| `--pfe-theme--font-size--heading--epsilon` | 16px |
+| `--pfe-theme--font-size--heading--zeta` | 14px |
+| `--pfe-theme--font-weight--light` | 300 |
+| `--pfe-theme--font-weight--normal` | 500 |
+| `--pfe-theme--font-weight--semi-bold` | 600 |
+| `--pfe-theme--font-weight--bold` | 700 |
+| `--pfe-theme--surface--border-width` | 1px |
+| `--pfe-theme--surface--border-width--heavy` | 4px |
+| `--pfe-theme--surface--border-style` | solid |
+| `--pfe-theme--surface--border-radius` | 3px |
+| `--pfe-theme--surface--border-width--active` | 3px |
+| `--pfe-theme--surface--border-width--color` | #151515 |
+| `--pfe-theme--icon-size` | 1em |
+| `--pfe-theme--ui--element--size` | 20px |
+| `--pfe-theme--ui--border-width` | 1px |
+| `--pfe-theme--ui--border-width--active` | 3px |
+| `--pfe-theme--ui--border-style` | solid |
+| `--pfe-theme--ui--border-radius` | 2px |
+| `--pfe-theme--ui--focus-outline-width` | 1px |
+| `--pfe-theme--ui--focus-outline-style` | solid |
+| `--pfe-theme--link-decoration` | none |
+| `--pfe-theme--link-decoration--hover` | underline |
+| `--pfe-theme--link-decoration--focus` | underline |
+| `--pfe-theme--link-decoration--visited` | none |
+| `--pfe-theme--link-decoration--on-dark` | none |
+| `--pfe-theme--link-decoration--hover--on-dark` | underline |
+| `--pfe-theme--link-decoration--focus--on-dark` | underline |
+| `--pfe-theme--link-decoration--visited--on-dark` | none |
+| `--pfe-theme--link-decoration--on-saturated` | underline |
+| `--pfe-theme--link-decoration--hover--on-saturated` | underline |
+| `--pfe-theme--link-decoration--focus--on-saturated` | underline |
+| `--pfe-theme--link-decoration--visited--on-saturated` | underline |
+| `--pfe-theme--grid-breakpoint--xs` | 0 |
+| `--pfe-theme--grid-breakpoint--sm` | 576px |
+| `--pfe-theme--grid-breakpoint--md` | 768px |
+| `--pfe-theme--grid-breakpoint--lg` | 992px |
+| `--pfe-theme--grid-breakpoint--xl` | 1200px |
+| `--pfe-theme--box-shadow--sm` | 0 0.0625rem 0.125rem 0 rgba(3, 3, 3, 0.2) |
+| `--pfe-theme--box-shadow--md` | 0 0.125rem 0.0625rem 0.0625rem rgba(3, 3, 3, 0.12), 0 0.25rem 0.6875rem 0.375rem rgba(3, 3, 3, 0.05) |
+| `--pfe-theme--box-shadow--lg` | 0 0.1875rem 0.4375rem 0.1875rem rgba(3, 3, 3, 0.13), 0 0.6875rem 1.5rem 1rem rgba(3, 3, 3, 0.12) |
+| `--pfe-theme--box-shadow--inset` | inset 0 0 0.625rem 0 #fafafa |
+| `--pfe-theme--animation-speed` | 0.3s |
+| `--pfe-theme--animation-timing` | cubic-bezier(0.465, 0.183, 0.153, 0.946) |
+| `--pfe-theme--opacity` | 0.09 |
+| `--pfe-theme--modal` | 99 |
```

Comment:
@castastrophe should these last 4 variables have z-index in the name? in the redhat-theme/theme.css we have:

```
  --pfe-theme--zindex--modal: 1001;
  --pfe-theme--zindex--navigation: 103;
  --pfe-theme--zindex--subnavigation: 95;
  --pfe-theme--zindex--content: 0;
```

Reply:
Yes, they should

Reply:
opened a separate issue for this: https://github.com/patternfly/patternfly-elements/issues/1034